t/ingredients_processing.t

```diff
@@ -194,24 +194,9 @@ my @tests = (
 				'id' => 'en:shallot',
 				'processing' => 'en:pureed',
 				'text' => 'Schalotte'
-				}
```

Comment:
can you put those tests back? with the new built ingredients taxonomy, they should work

Reply:
I added them in